File: 프로그래머스/0/120826. 특정 문자 제거하기/특정 문자 제거하기.java
import java.util.*;

class Solution {
    public String solution(String my_string, String letter) {
        StringBuilder sb = new StringBuilder();
        
        for(String str : my_string.split("")) {
            if(!str.equals(letter)) {
                sb.append(str);
            }
        }
        
        return sb.toString();
    }
}

// 다른 풀이
return my_string.replace(letter, "");
// .replace(타겟, 대체할 문자) : 처음 발견한 문자를 치환 후 반환
// .replaceAll(타겟, 대체할 문자) : 발견한 모든 문자를 치환 후 반환

File: 프로그래머스/0/120826. 특정 문자 제거하기/특정 문자 제거하기.py
def solution(my_string, letter):
    return my_string.replace(letter, '')

# .replace(타겟, 변경할 문자 [, 변경할 횟수]) : 문자열을 변경하는 함수
# 횟수 생략 시 문자열 전체를 변경

# 다른 답안 1
answer = ''
for string in my_string:
    if string != letter:
        answer += string
return answer

# 다른 답안 2
return ''.join([c for c in my_string if c != letter])
